src/code/views/fullscreen-button.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import * as screenfull from "screenfull";
+
+// import "./fullscreen-button.sass";
+
+interface IProps {
+  onFullscreenChange?: (isFullscreen: boolean) => void;
+}
+
+interface IState {
+  isFullscreen: boolean;
+}
+
+export class FullScreenButton extends React.Component<IProps, IState> {
+
+  public state = {
+    isFullscreen: false
+  };
+
+  public componentDidMount() {
+    if (screenfull.isEnabled) {
+      screenfull.on("change", () => {
```

Comment:
Looks like you need to add a `componentWillUnmount` method and call `screenfull.off("change", ...)` and move the callback out into the class.

https://github.com/sindresorhus/screenfull.js/#detect-fullscreen-change

Reply:
:+1: